src/slices/QuoteCustomerCard/QuoteCustomerCard.tsx

```diff
@@ -0,0 +1,60 @@
+import { Container, Heading, Button, Link, Box, Text } from 'boemly';
+import React, { useContext } from 'react';
+import { StrapiCustomerStory, strapiMediaUrl } from '../..';
+import Image from 'next/image';
+import { IntlContext } from '../../components/ContextProvider';
+
+export interface QuoteCustomerCardProps {
+  customerStory: StrapiCustomerStory;
+}
+export const QuoteCustomerCard = ({
```

Comment:
```suggestion
export const CustomerQuoteCard = ({
```

What do you think? Just a suggestion.